### PR TITLE
Removes the soul from chaplain.dm (Removes banbait religions)

### PR DESCRIPTION
**Pull request by Sinestia**
## About The Pull Request

Deletes the religions Ruko cleaned out from 
https://wiki.beestation13.com/view/Religion

which could get you banned if you used them already.

No more brain damage religion either. 

## Why It's Good For The Game

Banbait bad, with the current server rules there's no good reason to pick any of those religions. 

## Changelog
:cl:Froststahr
del: A few religions whose use would result in risking a ban have been deleted. 
/:cl:

### CODE REVIEW

**Comment by jupyterkat** [comment]
where’s the fun

**Comment by Rukofamicom** [comment]
Headmin approved, for whatever it's worth. 

I had just ripped several of them off of the wiki yesterday.

**Comment by utilbird** [comment]
the absolute state

**Comment by Sinestia** [comment]
> 
> 
> the absolute state

I will miss Guys Gone Wild 😔

**Comment by ghost** [comment]
Guys gone wild is debatable but toolboxia? wtf

**Comment by DungeonPenguin** [comment]
bee being fucking dumb yet again

**Comment by mudzbe** [comment]
Fumno

**Comment by FriendlyContractor** [comment]
I think they are in spirit of the silly game SS13 is and fail to see the problem with them. Im a little worried this PR might be a slippery slope to removing humor i belive most of us enjoy.

**Comment by williamcll** [comment]
> Guys gone wild is debatable but toolboxia? wtf

Sadly greytiding is not allowed on MRP

File: code/modules/jobs/job_types/chaplain.dm [hunk 1]
```diff
@@ -98,10 +93,6 @@
 			B.desc = "Happy, Full, Clean. Live it and give it."
 		if("subgenius")
 			B.name = "Book of the SubGenius"
-		if("toolboxia","greytide")
-			B.name = pick("Toolbox Manifesto","iGlove Assistants")
```
Comment (ike709) [inline code/modules/jobs/job_types/chaplain.dm]: I liked this one :(

Comment (Sinestia) [reply code/modules/jobs/job_types/chaplain.dm]: I could just remove homosexuality and the brain damage one, the general principle was that things that'd get you banned shouldn't be in this code, but admittedly the last two aren't very significant.

Comment (PowerfulBacon) [reply code/modules/jobs/job_types/chaplain.dm]: yea

Comment (Archanial) [reply code/modules/jobs/job_types/chaplain.dm]: greytide

**Review by Archanial — DISMISSED** [review]
Be sure to edit wiki

File: code/modules/jobs/job_types/chaplain.dm (hunk 1 above)
Comment (Sinestia) [reply code/modules/jobs/job_types/chaplain.dm]: Went with this

**Comment by utilbird** [comment]
bee homophobic 😢

**Pull request merged**